client/views/app/room.coffee

```diff
@@ -281,18 +281,10 @@ Template.room.helpers
 		# this is to make "bannerData" itself reactive by having it depend directly on the room data.
 		# Then, since that data gets synchronized with the server, the template will be reprocessed
 		# when the data changes.
-		accessPermissions = ChatRoom.findOne(this._id)?.accessPermissions || []
-		Template.instance().updateBannerData(accessPermissions)
-		return Template.instance().bannerData
-
-	# For helpers "classificationId" and "securityBannerText", "this" refers to what is returned
-	# from "bannerData"
-	classificationId: ->
-		return this.get 'classificationId'
-
-	securityBannerText: ->
-		return this.get 'text'
-		
+		roomData = Session.get('roomData' + this._id)
+		Template.instance().accessPermissions.set roomData?.accessPermissions
+		return Template.instance().accessPermissions
```

Comment:
I know it's not really the purpose of this pull request, but would it make sense to just let the 'securityBanner' template access this session variable directly (it's reactive)? Or does it help make the code clearer by specifying exactly the data context getting passed to the 'securityBanner' template (the permission list)?